USE-CASES/W3C-LBD-CG.md

```diff
@@ -284,7 +159,7 @@ Example of a temperature setpoint actuator:
     ],
     "title": "Kitchen-tempS-Actuator",
     "description": "Kitchen Temperature Setpoint Actuator",
-    "@type": ["sosa:Actuator", "dog:ThermostaticRadiatorValve", "bot:element"],
+    "@type": ["sosa:Actuator", "brick:Zone_Air_Temperature_Setpoint", "bot:element"],
```

Comment:
should add `"brick": "https://brickschema.org/schema/1.1/Brick#"` to `@context` and remove `"dog"`